### DISCUSSION TITLE
VCチャンネルに自動で参加させたいです

### DISCUSSION THREAD

**Discussion by AzukiShiratama** [ciffelia/koe#95] [Ideas]
初めまして。

こちらのBOTの導入を試しているのですが、誰かがVCチャンネルに参加した時に、こちらのBOTも自動的にjoinさせる方法はありませんでしょうか？
もしアップデートで対応していただけるなら大変ありがたいです。

よろしくお願いいたします。

**Comment by ciffelia** [comment]
現在のところはそのような機能はありません。お約束はできませんが、技術的には可能なのでいつか実装するかもしれません。
どうしても必要であれば、フォークしてご自身で実装していただければと思います（PRを送っていただければ機能のメンテナンスはするつもりです）。
